packages/govern-create/tasks/sequences/deploy-govern.ts

```diff
@@ -0,0 +1,91 @@
+import { task } from 'hardhat/config'
+import { verifyContract } from '../../helpers/etherscan-verification'
+import {
+  buildName,
+  getContract,
+  getGovernBaseFactory,
+  getGovernRegistry,
+  setHRE,
+  NO_TOKEN,
+  ZERO_ADDR,
+} from '../../helpers/helpers'
+import { logDeploy, logInfo } from '../../helpers/logger'
+import { eContractid } from '../../helpers/types'
+import { GovernBaseFactory } from '../../typechain'
+
+task('deploy-govern', 'Deploys a Govern instance')
+  .addOptionalParam('factory', 'GovernBaseFactory address')
+  .addOptionalParam('useProxies', 'Whether to deploy govern with proxies')
+  .addOptionalParam('name', 'DAO name (must be unique at GovernRegistry level)')
+  .addFlag('verify', 'Verify the contracts via Etherscan API')
+  .setAction(
+    async (
+      {
+        factory,
+        useProxies = true,
+        name,
+        verify,
+        token = NO_TOKEN,
+        tokenName = name,
+        tokenSymbol = 'GOV',
+      },
+      HRE
+    ) => {
+      setHRE(HRE)
+      const currentNetwork = HRE.network.name
+
+      name = buildName(name)
+
+      const registry = await getGovernRegistry()
+      const baseFactory = factory
+        ? await getContract<GovernBaseFactory>(
+            eContractid.GovernBaseFactory,
+            factory
+          )
+        : await getGovernBaseFactory()
+
+      const tx = await baseFactory.newGovernWithoutConfig(
+        name,
+        token,
+        tokenName || name,
+        tokenSymbol,
+        useProxies,
+        {
+          gasLimit: useProxies ? 2e6 : 9e6,
```

Comment:
A cool fancy way to write numbers but would recommend to write them out and probably have those in a configuration or at least as process env.